src/pages/Map/MapBtn.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState } from 'react';
+import MainMap from './MainMap';
+import HostStore from './Mapcontext';
+import variables from '../../styles/variables';
+import styled from 'styled-components';
+
+const MapBtn = () => {
+  const [display, setDisplay] = useState(false);
+  const modal = () => {
+    if (!display) {
+      setDisplay(true);
+    } else {
+      setDisplay(false);
+    }
+  };
+
+  return (
+    <HostStore>
+      {display ? <MainMap display={display} /> : null}
```

Comment:
display는 boolean값을 다루기 떄문에 
```suggestion
      {display && <MainMap display={display} /> }
```
은 어때요?